sql/dados.sql

```diff
@@ -0,0 +1,554 @@
+       ('Fraternidade das Dimensões', 'Zentar'),
+       ('Congregação Estelar', 'Korvax'),
+       ('Confederação das Esferas', 'Yolthar'),
+       ('União dos Mundos Unidos', 'Florin'),
+       ('Pacto dos Astros', 'Sylthor'),
+       ('Assembleia das Constelações', 'Astron'),
+       ('Legião dos Viajantes do Espaço', 'Naxor'),
+       ('Reinos Intergalácticos', 'Kryllar'),
+       ('Sociedade das Estrelas', 'Zephyr'),
+       ('Coletivo das Nebulosas', 'Eldar'),
+       ('Legado dos Antigos', 'Varkor'),
+       ('Reinos da Fronteira', 'Zyndari'),
+       ('Círculo das Luas Lunares', 'Graxion'),
+       ('Conclave Estelar', 'Xyrron'),
+       ('Ordem dos Guardiões da Galáxia', 'Krothar'),
+       ('Liga dos Exploradores Cósmicos', 'Zelthar'),
+       ('Sindicato das Constelações', 'Zylok'),
+       ('União Astral', 'Thyrrax'),
+       ('Pacto Intergaláctico', 'Xylos'),
+       ('Ordem das Estrelas Luminosas', 'Vylkor'),
+       ('Aliança Celestial', 'Grynthor'),
+       ('Concílio das Esferas Cósmicas', 'Zorgon'),
+       ('Liga dos Planetas Livres', 'Xenoth'),
+       ('Consórcio Interplanetário', 'Drakar'),
+       ('Tribo das Nebulosas Eternas', 'Nelkari'),
+       ('Soberanos da Galáxia', 'Lyrion'),
+       ('Irmandade Estelar', 'Vrillar'),
+       ('Conglomerado Intergaláctico', 'Gorix'),
+       ('Fraternidade das Dimensões', 'Zygoran'),
+       ('Congregação Estelar', 'Ogriak'),
+       ('Confederação das Esferas', 'Zentar'),
+       ('Pacto dos Astros', 'Kryllar'),
+       ('Assembleia das Constelações', 'Zephyr'),
+       ('Legião dos Viajantes do Espaço', 'Eldar'),
+       ('Reinos Intergalácticos', 'Varkor'),
+       ('Sociedade das Estrelas', 'Zyndari'),
+       ('Coletivo das Nebulosas', 'Graxion'),
+       ('Legado dos Antigos', 'Xyrron'),
+       ('Reinos da Fronteira', 'Krothar'),
+       ('Círculo das Luas Lunares', 'Zelthar'),
+       ('Conclave Estelar', 'Zylok'),
+       ('Ordem dos Guardiões da Galáxia', 'Thyrrax'),
+       ('Liga dos Exploradores Cósmicos', 'Xylos'),
+       ('Sindicato das Constelações', 'Vylkor'),
+       ('União Astral', 'Grynthor'),
+       ('Pacto Intergaláctico', 'Zorgon'),
+       ('Ordem das Estrelas Luminosas', 'Xenoth'),
+       ('Aliança Celestial', 'Drakar'),
+       ('Concílio das Esferas Cósmicas', 'Nelkari'),
+       ('Liga dos Planetas Livres', 'Lyrion'),
+       ('Consórcio Interplanetário', 'Vrillar'),
+       ('Tribo das Nebulosas Eternas', 'Gorix'),
+       ('Soberanos da Galáxia', 'Zygoran'),
+       ('Irmandade Estelar', 'Ogriak'),
+       ('Conglomerado Intergaláctico', 'Zentar'),
+       ('Fraternidade das Dimensões', 'Korvax'),
+       ('Congregação Estelar', 'Yolthar'),
+       ('Confederação das Esferas', 'Florin'),
+       ('União dos Mundos Unidos', 'Sylthor'),
+       ('Pacto dos Astros', 'Astron'),
+       ('Assembleia das Constelações', 'Naxor');
+
+
+INSERT INTO COLONIA (ID, GALAXIA, NOME_PLANETA, FACÇÃO_NOME, DH_INICIO, DH_FIM, NOME)
```

Comment:
Tirar o acento aqui tbm
```suggestion
INSERT INTO COLONIA (ID, GALAXIA, NOME_PLANETA, FACCAO_NOME, DH_INICIO, DH_FIM, NOME)
```